OpenRA.Mods.Common/Widgets/RadarWidget.cs

```diff
@@ -22,6 +22,9 @@ namespace OpenRA.Mods.Common.Widgets
 {
 	public sealed class RadarWidget : Widget, IDisposable
 	{
+		static readonly int ColorFog = Color.FromArgb(128, Color.Black).ToArgb();
+		static readonly int ColorShroud = Color.Black.ToArgb();
```

Comment:
I suggest `public readonly` because why not make this .yaml configurable?

Reply:
could be a seperate future pull request?